<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditDialog</class>
 <widget class="QDialog" name="EditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #17191b;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">color: #fbfafa;
border: 1px solid #2b2b2b;
border-radius: 12px;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="styleSheet">
         <string notr="true">color: #fbfafa;
background: #191a1f;</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="productEditLabel">
           <property name="font">
            <font>
             <family>Carlito</family>
             <pointsize>16</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #fbfafa;
border: none;
border-radius: 0;
padding: 15px 0 15px 5px;</string>
           </property>
           <property name="text">
            <string>PRODUTOS/S:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="productEditInput">
           <property name="font">
            <font>
             <family>Carlito</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #fbfafa;
border: none;
border-radius: 0;
padding: 15px 0 15px 5px;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="styleSheet">
         <string notr="true">color: #fbfafa;
background: #191a1f;</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="weighEdittLabel">
           <property name="font">
            <font>
             <family>Carlito</family>
             <pointsize>16</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #fbfafa;
border: none;
border-radius: 0;
padding: 15px 0 15px 5px;</string>
           </property>
           <property name="text">
            <string>PESO:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="weightEditInput">
           <property name="font">
            <font>
             <family>Carlito</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #fbfafa;
border: none;
border-radius: 0;
padding: 15px 0 15px 5px;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="font">
        <font>
         <family>Carlito</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background: #25c665;
color: #fbfafa;
border-radius: 12px;
padding: 20px;
}

QPushButton:hover 
{
    background: #2cd870;
}</string>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="font">
        <font>
         <family>Carlito</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background: #25c665;
color: #fbfafa;
border-radius: 12px;
padding: 20px;
}

QPushButton:hover 
{
    background: #2cd870;
}</string>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
